cypress/support/commands.js

```diff
@@ -24,47 +24,56 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
+// Логін користувача через API
+// Реєстрація користувача через API
+// Реєстрація користувача через API
 
+Cypress.Commands.add('registerUser', (email, username, password) => {
```

Comment:
It's good to see that you're using Cypress's custom commands to extend its functionality. This makes your tests more readable and maintainable.